lib/reportDisables.js

```diff
@@ -43,21 +35,20 @@ module.exports = function (results) {
 			rangeData[rule].forEach((range) => {
 				if (!reportDisablesForRule(_.get(config, ['rules', rule], []))) return;
 
-				reported.ranges.push({
-					rule,
-					start: range.start,
-					end: range.end,
-					unusedRule: rule,
+				// If the comment doesn't have a location, we can't report a useful error.
+				// In practice we expect all comments to have locations, though.
+				if (!range.comment.source || !range.comment.source.start) return;
+
+				result.warnings.push({
```

Comment:
maybe we need sort result by line? previously there was some sorting

Reply:
I believe lints are automatically sorted (since they're all initially appended to the lint list in the order the lints are run, they'd have to be). The only reason this had to manually sort was that it wasn't being treated as a lint.

I checked manually and it does look like these lints are already being sorted appropriately relative to existing lints in the report.